packages/desktop/src/renderer/sagas/modals/modals.types.ts

```diff
@@ -21,4 +21,6 @@ export enum ModalName {
   searchChannelModal = 'searchChannelModal',
   warningModal = 'warningModal',
   channelCreationModal = 'channelCreationModal',
+  duplicateModal = 'duplicateModal',
```

Comment:
I'm missing the information what specifically is duplicated. I think naming it `duplicatedUserModal` or `duplicatedUsernameModal` would be better